pxd.h

```diff
@@ -86,6 +86,7 @@ enum pxd_opcode {
 	PXD_FALLBACK_TO_KERNEL,   /**< Fallback requests suspend IO and send in a marker req
 						  from kernel on a suspended device */
 	PXD_EXPORT_DEV,     /**< export the attached device to the kernel */
+	PXD_WRITE_ZEROES,   /**< write_zeroes operation */
```

Comment:
need to add features export for new op code.
So px-storage knows that this driver supports write zeroes control.